packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -609,7 +617,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
         { ...splitButtonProps }
         data-ktp-execute-target={ keytipAttributes['data-ktp-execute-target'] }
         onMouseDown={ this._onMouseDown }
-        tabIndex={ -1 }
+        tabIndex={ 0 }
```

Comment:
Why is this being set to tabindex 0?

Reply:
From what I've learned so far, this is the only way to override focus zones' automatic behavior for splitbutton to place focus on it - if there are other ways I'm unaware of, please let me know & I can try them out!

Reply:
The sub portions of a splitButton should never be individually focusable

Reply:
The sub portions are not individually focusable, it's the whole button that get focus

Reply:
This is for the splitButton menu button portion, it should always have a tabindex of -1